Gems/EMotionFX/Code/Source/Editor/Plugins/Ragdoll/ColliderCapsuleManipulators.cpp

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzToolsFramework/ViewportSelection/EditorSelectionUtil.h>
+#include <EMotionFX/CommandSystem/Source/ColliderCommands.h>
+#include <EMotionFX/Tools/EMotionStudio/EMStudioSDK/Source/RenderPlugin/ViewportPluginBus.h>
+#include <EMotionStudio/EMStudioSDK/Source/EMStudioManager.h>
+#include <Editor/ColliderContainerWidget.h>
+#include <Editor/Plugins/Ragdoll/ColliderCapsuleManipulators.h>
+
+namespace EMotionFX
+{
+    Physics::CapsuleShapeConfiguration* GetCapsuleShapeConfiguration(PhysicsSetupManipulatorData& physicsSetupManipulatorData)
+    {
+        if (!physicsSetupManipulatorData.HasCapsuleCollider())
+        {
+            return nullptr;
+        }
+
+        return azdynamic_cast<Physics::CapsuleShapeConfiguration*>(
+            physicsSetupManipulatorData.m_colliderNodeConfiguration->m_shapes[0].second.get());
+    }
+
+    Physics::CapsuleShapeConfiguration* GetCapsuleShapeConfiguration(const PhysicsSetupManipulatorData& physicsSetupManipulatorData)
+    {
+        if (!physicsSetupManipulatorData.HasCapsuleCollider())
+        {
+            return nullptr;
+        }
+
+        return azdynamic_cast<Physics::CapsuleShapeConfiguration*>(
+            physicsSetupManipulatorData.m_colliderNodeConfiguration->m_shapes[0].second.get());
+    }
+
+    ColliderCapsuleManipulators::ColliderCapsuleManipulators()
+    {
+        m_adjustColliderCallback = AZStd::make_unique<PhysicsSetupManipulatorCommandCallback>(this, false);
+        EMStudio::GetCommandManager()->RegisterCommandCallback("AdjustCollider", m_adjustColliderCallback.get());
+    }
+
+    ColliderCapsuleManipulators::~ColliderCapsuleManipulators()
+    {
+        EMStudio::GetCommandManager()->RemoveCommandCallback(m_adjustColliderCallback.get(), false);
+    }
+
+    void ColliderCapsuleManipulators::Setup(PhysicsSetupManipulatorData& physicsSetupManipulatorData)
+    {
+        m_physicsSetupManipulatorData = physicsSetupManipulatorData;
+
+        if (!m_physicsSetupManipulatorData.HasColliders())
```

Comment:
question: since this is `ColliderCapsuleManipulators` class, shouldn't all the `HasColliders` checks in this file be `HasCapsuleCollider` instead?